.changeset/rich-jars-run.md

```diff
@@ -0,0 +1,7 @@
+---
+'@primer/react': patch
```

Comment:
this could be seen as more than a patch, but I don't believe any code relying on portal should break, as it should only _start_ not throwing in ssr, and otherwise be consistent with the current impl